src/imageops/colorops.rs

```diff
@@ -125,18 +127,21 @@ where
 
     let percent = ((100.0 + contrast) / 100.0).powi(2);
 
-    let mut image = RefCell::new(image);
-    image.borrow().pixels().map(|(x, y, mut pixel)| {
-        let f = pixel.map(|b| {
-            let c: f32 = NumCast::from(b).unwrap();
+    let mut image = image;
+    let image = addr_of_mut!(image);
+    unsafe {
```

Comment:
No. This does not come with safety qualifications, and the TODO was looking for a _better_ way. `unsafe` is not a better way. Just leave it as it was.

Reply:
Okay, I leave it as it was where the borrowing is a problem.